pybaseball/statcast_fielding.py

```diff
@@ -8,12 +8,12 @@
 from .utils import norm_positions, sanitize_statcast_columns
 
 @cache.df_cache()
-def statcast_outs_above_average(year: int, pos: Union[int, str], min_att: Union[int, str] = "q") -> pd.DataFrame:
+def statcast_outs_above_average(year: int, pos: Union[int, str], view: str = "Fielder", min_att: Union[int, str] = "q") -> pd.DataFrame:
 	pos = norm_positions(pos)
```

Comment:
We should have a docstring here in the first place, but it would be nice to include the options for `view` in that docstring. If you could start with the following and add the options, that would be awesome.

```
	"""Scrapes outs above average from baseball savant for a given year and position

	Args:
		year (int): Season to pull
		pos (Union[int, str]): Numerical position (e.g. 3 for 1B, 4 for 2B). Catchers not supported
		min_att (Union[int, str], optional): Integer number of attempts required or "q" for qualified. 
			Defaults to "q".

	Raises:
		ValueError: Failure if catcher is passed

	Returns:
		pd.DataFrame: Dataframe of defensive OAA for the given year and position for players who have met
			the given threshold
	"""
```